src/types/ServiceListComponent.ts

```diff
@@ -26,3 +26,9 @@ export interface ServiceItem {
 }
 
 export const IstioLogo = require('../assets/img/istio-logo.svg');
+
+export type SortField = {
```

Comment:
As a rule of thumb if you are going to export it then it should probably be an `interface` instead of `type`